docs/rc.xml.all

```diff
@@ -568,6 +567,7 @@
       <clickMethod></clickMethod>
       <sendEventsMode></sendEventsMode>
       <calibrationMatrix></calibrationMatrix>
+      <scrollFactor>1.0</scrollFactor>
```

Comment:
The format could be additionally added to the comment above for completeness.

Reply:
Thanks. Fixed.